platforms/core-runtime/service-provider/build.gradle.kts

```diff
@@ -0,0 +1,9 @@
+plugins {
+    id("gradlebuild.distribution.implementation-java")
+}
+
+description = "An API for providing internal services for Gradle modules"
+
+dependencies {
+    api(projects.javaLanguageExtensions)
```

Comment:
❓ How come we don't need `libs.inject`?

Reply:
It's not required by the classes currently moved into this new module